Cleanup codestarter test

turing85 opened the pull request:
- Remove unnecessary modifiers
- Unify  line-breaks 
- Unify indentation

turing85 committed:
Cleanup codestarter tests

- Remove unnecessary modifiers
- Unify line-breaks
- Unify indentation

iocanel reviewed (approved):
LGTM

turing85 commented:
uhm... I am a little bit at a loss here. Tests fail with
```
java.lang.AssertionError: 
[Snapshot is not matching (use -Dsnap to update it automatically): GrpcCodestartTest/testContent/src_test_java_ilove_quark_us_HelloGrpcServiceTest.java] 
```

Is this ignorable? If not: how to fix this?

ia3andy commented:
@turing85 you need to either:
- Fix the snapshots manually
- Run the test with -Dsnap to update them (as explained in the log)

The snapshots are here to make sure the result of generating app with your changes in the templates is as expected

gsmet commented:
I pushed an update to synchronize the snapshots.

gsmet commented on devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-routes-codestart/java/src/test/java/org/acme/MyDeclarativeRoutesTest.java:
```diff
@@ -8,15 +8,16 @@
-	public void testHelloRouteEndpointWithoutNameParameter() {
-		given().when().get("hello-route").then().statusCode(200).body(is("Hello Reactive Route !!"));
-	}
+		@Test
+		void testHelloRouteEndpointWithNameParameter() {
+				given().when().get("hello-route?name=Quarkus").then().statusCode(200).body(is("Hello Quarkus !!"));
+		}
+
+		@Test
+		void testHelloRouteEndpointWithoutNameParameter() {
+				given().when().get("hello-route").then().statusCode(200).body(is("Hello Reactive Route !!"));
+		}
```
I think you are using tabs here? At least, that's what GitHub seems to show? We should use 4 spaces instead.

turing85 replied:
Fixed

gsmet reviewed (dismissed):
I'm not sure about how two files are indented. Could you have a look?

turing85 commented:
> I'm not sure about how two files are indented. Could you have a look?

The tabs were there already. I replaced them. I also rebased onto the current main. @gsmet care to re-evalute the PR now?

By-the-by: I did a global search on tabs in `*.java` files. Let's say there were enough hits that intelliJ refused to replace them all in one go. Should we create an issue to replace all those tabs?

turing85 commented:
so... we wanna proceed with this PR or nah?

turing85 commented on integration-tests/devtools/src/test/resources/__snapshots__/ReactiveRoutesCodestartTest/testContent/src_test_java_ilove_quark_us_MyDeclarativeRoutesTest.java:
```diff
@@ -8,15 +8,16 @@
+		@Test
+		void testHelloRouteEndpointWithoutNameParameter() {
+				given().when().get("hello-route").then().statusCode(200).body(is("Hello Reactive Route !!"));
+		}
```
Fixed

ia3andy reviewed (approved):
LGTM
